cedar-policy-validator/src/schema.rs

```diff
@@ -1238,6 +1238,31 @@ impl cedar_policy_core::entities::Schema for ValidatorSchema {
             }
         }
     }
+
+    fn allowed_parent_types<'s>(
+        &'s self,
+        entity_type: &cedar_policy_core::ast::EntityType,
+    ) -> HashSet<cedar_policy_core::ast::EntityType> {
```

Comment:
I don't think this will let an action entity be a member of a any other entity.

It's a bit strange to need to handle action entities at all since they should be extracted from the schema instead of parsed separately. Not sure how best to deal with them.

Reply:
is it an error to include actions in `Entities`?  If so, schema-based parsing doesn't need to handle Action entities, and I think we're good here (except maybe improving the error message for that case)?

Reply:
I think it can be an error. You shouldn't be providing any actions that weren't defined in the schema.

Reply:
I think #73 will cover improving the error message in that case